styles/MyInvision/theme/collapsiblecategories.css

```diff
@@ -0,0 +1,49 @@
+/* Import base collapsible categories stylesheet */
+@import url('../../all/theme/collapsiblecategories.css');
+
+/* Styling overrides for MyInvision theme */
+
+
+/* Collapse button */
+.collapse-btn {
+	background-color: transparent;
+	position: relative;
```

Comment:
you only need background color and border.

Reply:
I don't really like the default look of the buttons so I've decided to make them look differently .

Reply:
you only need backgorund color and border because the rest of the lines in this rule are duplication of the original css. you should only have new CSS here.

Reply:
oh yeah sorry I misunderstood you sorry. Didn't realize that in the first place dumb me :)